### PR TITLE
Upgrade rollup and grunt to support better tree-shaking #1076

### PR DESCRIPTION
**Pull request by MSNev**

### CODE REVIEW

File: extensions/applicationinsights-debugplugin-js/package.json [hunk 1]
```diff
@@ -14,20 +14,21 @@
         "build": "npm run build:esm && npm run build:browser",
         "build:esm": "grunt debugplugin",
         "build:browser": "rollup -c",
-        "test": "grunt aitests",
+        "test": "",
```
Comment (xiao-lix) [inline extensions/applicationinsights-debugplugin-js/package.json:17]: Is this test removed?

File: AISKU/package.json [hunk 2]
```diff
@@ -29,23 +29,23 @@
...
-        "grunt-contrib-uglify": "3.1.0",
-        "grunt-ts": "^6.0.0-beta.15",
+        "grunt-ts": "^6.0.0-beta.22",
         "grunt-tslint": "^5.0.2",
         "globby": "^11.0.0",
-        "rollup": "^0.66.0",
-        "rollup-plugin-node-resolve": "^3.4.0",
-        "rollup-plugin-replace": "^2.1.0",
-        "rollup-plugin-uglify": "^6.0.0",
+        "@rollup/plugin-commonjs": "^15.1.0",
```
Comment (xiao-lix) [inline AISKU/package.json:45]: Are these 3 packages different? eg. `rollup-plugin-node-resolve` vs `@rollup/plugin-node-resolve`, is the package name changed?

File: tools/rollup-plugin-uglify3-js/rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    output: {
+      file: `${distPath}node/${outputName}.js`,
+      banner: banner,
+      format: "umd",
+      name: "Microsoft.ApplicationInsights-Rollup-Plugin-Uglify",
+      extend: true,
+      freeze: false,
+      sourcemap: false,
+      externalLiveBindings: false,
+      globals:[ 'UglifyJs ']
```
Comment (xiao-lix) [inline tools/rollup-plugin-uglify3-js/rollup.config.js:30]: Is the space here on purpose?

File: AISKU/package.json (hunk 2 above)
Comment (MSNev) [reply AISKU/package.json:45]: This is part of rollup moving their packages, so the latest version need to use the new package location

File: tools/rollup-plugin-uglify3-js/rollup.config.js (hunk 3 above)
Comment (MSNev) [reply tools/rollup-plugin-uglify3-js/rollup.config.js:30]: nice catch!

Comment (MSNev) [reply tools/rollup-plugin-uglify3-js/rollup.config.js]: Fixed

File: AISKU/package.json [hunk 4]
```diff
@@ -14,7 +14,7 @@
         "clean": "rm -rfv browser types dist-esm",
         "build": "npm run build:esm && npm run build:browser",
         "build:esm": "grunt aisku",
-        "build:browser": "rollup -c",
+        "build:browser": "rollup -c rollup.config.js",
```
Comment (kryalama) [inline AISKU/package.json:17]: nit: do we need this change?

File: AISKULight/package.json [hunk 5]
```diff
@@ -9,20 +9,24 @@
     "scripts": {
         "build": "npm run build:esm && npm run build:browser",
         "build:esm": "grunt aiskulite",
-        "build:browser": "rollup -c",
+        "build:browser": "rollup -c rollup.config.js",
```
Comment (kryalama) [inline AISKULight/package.json:12]: nit: do we need this changes?

**Review by kryalama — APPROVED** [review]

**Pull request merged**